README.md

```diff
@@ -631,13 +631,14 @@ communicate with the Conjur appliance.
 
```

Comment:
Following our earlier conversation, a more fitting title prior to reaching this section might be something like:

"Conjur Authenticator / Secretless Sidecar / Secrets Provider Injection Examples"

or any shorter version :)